content/articles/running-a-multi-container-springboot-postgresql-application-with-docker-compose/index.md

```diff
@@ -0,0 +1,207 @@
+Container technology has streamlined how we build, test and deploy software from our local environment to on-premise data centers and the cloud. With the benefit of building applications with container technology, manually starting and stopping each container becomes tedious as you build multi-container applications. 
+
+Docker Compose is a tool used in defining and running multi-container Docker applications. We use Docker Compose to replace the `run` commands that is repeatedly run on each container in a multi-container application. Using Docker Compose, we can simplify the running of multi-container applications to as little as two commands; `docker-compose up` and `docker-compose down`.
+
+### Goal
+
+In this tutorial, you’ll learn how Docker Compose makes it easier to configure and run multiple Docker containers in a local environment. For demo purposes, this tutorial will use a Spring Boot and a PostgreSQL container.
+
+### Prerequisites
+
+- Basic knowledge of Spring Boot, Git, and the terminal.
+- Basic understanding of Docker and Docker Compose installed - see guide [here](https://docs.docker.com/compose/install/).
+- A Java IDE - In this tutorial, we will use [IntelliJ Idea](https://www.jetbrains.com/idea/), but you can use any IDE of your choice.
+- Postman - download it [here](https://www.postman.com/downloads/)
+
+### Setting up a Spring Boot project
+
+To start, clone this Spring Boot project: https://github.com/Kikiodazie/BlogAPI from Github. The Spring Boot project is a blog REST API you will use to Create and Retrieve blog posts in the PostgreSQL database you’re going to create.
+
+If you are not familiar with how to clone a Github repository, [click here](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository-from-github/cloning-a-repository) for detailed instructions.
+
+The blog API has just the following endpoints:
+
+| **Endpoints** | **Functionality** |
+| ------------- | ----------------- |
+| POST /posts   | Create a new post |
+| GET /posts    | List all todos    |
+
+After cloning, in the project directory, go to the `application.properties` file in `src/main/resources/application.properties` directory to configure the API data source.
+
+
+![BlogAPI project directory structure showing application.properties file](/running-a-multi-container-springboot-postgresql-application-with-docker-compose/directory-structure.png)
+
+
+In the `application.properties` file add:
+
+```bash
+    spring.datasource.driverClassName=org.postgresql.Driver
+```
+
+The above configuration tells Spring Boot that the data source will be of PostgreSQL using PostgreSQL’s specific driver.
+
+Normally when setting up a spring boot project to connect to a data source, we use similar to in the `application.properties` file:
+
```

Comment:
This sentence is not clear...."we use similar to in the...." is there something missing?

Reply:
Ohhh. will look into it right away.

Reply:
Done. Do check it out.